cumulus/client/cirrus-executor/src/bundle_processor.rs

```diff
@@ -463,18 +481,17 @@ where
 			.extract_receipts(&BlockId::Hash(primary_hash), extrinsics.clone())?;
 
 		for signed_receipt in signed_receipts.iter() {
-			match aux_schema::load_execution_receipt::<
+			match crate::aux_schema::load_execution_receipt::<
 				_,
 				Block::Hash,
 				NumberFor<PBlock>,
 				PBlock::Hash,
 			>(&*self.client, signed_receipt.execution_receipt.secondary_hash)?
 			{
 				Some(local_receipt) => {
-					if let Some(trace_mismatch_index) = crate::find_trace_mismatch(
-						&local_receipt,
-						&signed_receipt.execution_receipt,
-					) {
+					if let Some(trace_mismatch_index) =
+						find_trace_mismatch(&local_receipt, &signed_receipt.execution_receipt)
```

Comment:
It is a bit inconsistent all over the place. You removed `crate::` from one and added `crate::` to another. Looks weird, I don't understand why.

Reply:
Hmm, no particular reason, using `crate::aux_schema::*` was probably for indicating it's a database-related operation more explicitly.